app/src/main/java/com/sma2/sma2/DataAccess/ExerciseDA.java

```diff
@@ -2,64 +2,122 @@
 
 import android.support.annotation.IdRes;
 
+import com.sma2.sma2.ExerciseLogic.Exercise;
+
 import org.greenrobot.greendao.annotation.Entity;
 import org.greenrobot.greendao.annotation.Id;
 import org.greenrobot.greendao.annotation.Keep;
 import org.greenrobot.greendao.annotation.NotNull;
 import org.greenrobot.greendao.annotation.Generated;
+import org.greenrobot.greendao.annotation.Property;
 
 @Entity
 public class ExerciseDA {
     @Id private  Long id;
 
     @NotNull
     private String exerciseName;
-    private String examplePath;
-    private String instructionsPath;
+    private String instructionTextPath;
+    private String instructionVideoPath;
+    private String fragmentClassName;
     private String exerciseType;
+    @Property
+    private String description;
```

Comment:
I think we should be coherent with the naming and call these two shortDescription and shortInstructions like in the dataclass